package.json

```diff
@@ -18,6 +18,7 @@
   "scripts": {
     "clean": "rm -rf dist",
     "build": "npm run clean && tsc && cp -r templates dist/templates && cp -r examples dist/examples",
+    "publish-to-npm": "./cleanup_for_publish.sh && pnpm i && pnpm build && npm publish",
```

Comment:
we use `npm` as the package manage, all of our scripts use `npm`
```suggestion
    "publish-to-npm": "./cleanup_for_publish.sh && npm install && npm run build && npm publish",
```

Reply:
hmm i thought for CAD itself, we use pnpm?

Reply:
then ill remove the pnpm lock?